### ISSUE TITLE
Maya: GUI elements inconsistency

### ISSUE THREAD

**Issue by m-u-r-p-h-y** [pypeclub/OpenPype#2831]
There is a bit of inconsistency in OpenPype's Maya GUI elements
It can be quite frustrating for a new user to navigate through different parts of OP.

To help visually we can use the same icons in OpenPype menu as we use in side-shelf and match the order.  (Later we can open discussion about the order itself as it can be improved as well and should follow the workflow better)

 
![mayaMenuOP](https://user-images.githubusercontent.com/352795/156392249-81f8c2a4-2b91-4e25-92a3-ba4e4c639540.png)

We should also unify the naming as it differs and makes it even more confusing

| side-shelf | OpenPypeMenu     |      Tool Title bar |
| ------------ | ---------------- | ----------------- |
| Look Manager | Look assigner...  | Look Manager |
| Loader | Load... | Asset Loader |
| Inventory | Manage... | Scene Inventory |

Haven't looked at other host implementations, but I suppose it will be similar

**Comment by BigRoy** [comment]
I think personally the icons would also benefit from dropping the gradient and have a single color tint. Overall that just feels easier to the eye - or have some more definition on the icons like the Tray "P" icon.

**Comment by mkolar** [comment]
I agree. Unifying the colours for now and making sure they are used everywhere would be a good start

I'd also replace the Pyblish icon with something that fits OP visual, the funnel is not particularly recognisable as publishing symbol by most people I talk to.

**Comment by BigRoy** [comment]
@mkolar Would be good to discuss this naming convention internally at Ynput and then report back with the preferred naming convention we should apply across hosts and labels to be consistent.

> I'd also replace the Pyblish icon with something that fits OP visual, the funnel is not particularly recognisable as publishing symbol by most people I talk to.

I'd say this could be a separate issue, nonetheless - if you have anything in mind already that'd be great. Otherwise maybe a dedicated issue of designing maybe icons for certain tools (if needed) could be worth it.

**Comment by BigRoy** [comment]
> I'd also replace the Pyblish icon with something that fits OP visual, the funnel is not particularly recognisable as publishing symbol by most people I talk to.

@mkolar I was just looking for this and noticed there's still no such icon in [resources icons](https://github.com/ynput/OpenPype/tree/ac8d9865f5902f01ea9d6750b09b82a2995d54dc/openpype/resources/icons).
